internal/attachment/repository/mysql/mysql_attachment_test.go

```diff
@@ -0,0 +1,33 @@
+package mysql_test
+
+import (
+	"context"
+	"testing"
+
+	mysqlrepo "github.com/meroedu/meroedu/internal/attachment/repository/mysql"
+	"github.com/meroedu/meroedu/internal/domain"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/DATA-DOG/go-sqlmock.v1"
```

Comment:
Please sort these imported packages, separate it by internal and external dependencies.